demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/CarReviewsFAQsModulePage.java

```diff
@@ -0,0 +1,83 @@
+    private WebElement lblsecondAnswer = driver.findElement(By.cssSelector(".css-13q1jog-GridContainer.eds0yfe0>div:nth-child(2)>div"));
+    private WebElement lblthirdQuestion = driver.findElement(By.xpath("//h4[normalize-space()='What is the MPG for a 2012 Audi A3?']"));
+    private WebElement lblthirdAnswer = driver.findElement(By.xpath("//div[normalize-space()='The 2012 Audi A3 gets 22 mpg in the city and 28 mpg on the highway.']"));
+    
+    public boolean isUrlCorrect(String year, String make, String model){
+        String expectedUrl = "https://staging.kbb.com/"+  make.toLowerCase() +"/"+ model.toLowerCase() +"/"+ year +"/";
+        if(driver.getCurrentUrl().contentEquals(expectedUrl)){
+            return true;
+        }else{
+            return false;
+        }
+    }
+
+    public boolean isLblSelectedCarCorrect(String year, String make, String model){
+        String expectedLabel = year + " " + make + " " + model;
+        if (Integer.parseInt(year) < 2021) {
+            expectedLabel = "Used " + expectedLabel;
+        }
+        if(isElementTextEqualExpectedText(lblSelectedCar, expectedLabel)){
+            return true;
+        }else{
+            return false;
+        }
+    }
+
+    public boolean assertScopeIsDefaultUIOfFAQsSectionDiplayedCorrect(){
+        keyword.scrollToElement(lblFAQs);
+        if(isElementDisplayed(lblfirstQuestion) == true
+        && isElementDisplayed(lblfirstAnswer) == true
+        && isElementDisplayed(lblsecondQuestion) == true
+        && isElementDisplayed(lblsecondAnswer) == true
+        && isElementDisplayed(lblthirdQuestion) == true
+        && isElementDisplayed(lblthirdAnswer) == true
+        && isElementTextEqualExpectedText(lblfirstQuestion, firstQuestion) == true
+        && isElementTextEqualExpectedText(lblfirstAnswer, firstAnswer) == true
+        && isElementTextEqualExpectedText(lblsecondQuestion, secondQuestion) == true
+        && isElementTextEqualExpectedText(lblsecondAnswer, secondAnswer) == true
+        && isElementTextEqualExpectedText(lblthirdQuestion, thirdQuestion) == true
+        && isElementTextEqualExpectedText(lblthirdAnswer, thirdAnswer) == true)
+        {
+            return true;
+        }else{
+            return false;
+        }
+    }
+
+    public boolean assertScopeAreUrlAndLblSelectedCarCorrect(String year, String make, String model){
+        if(isLblSelectedCarCorrect(year, make, model) == true && isUrlCorrect(year, make, model) == true){
+            return true;
+        }else{
+            return false;
+        }
+    }
+
+    public String firstQuestion = "Is 2012 a good year for a Audi A3?";
+    public String firstAnswer = "According to the experts at Kelley Blue Book, the 2012 Audi A3 scores 3.8 out of 5 stars. Owners of the vehicle give it 4.6 out of 5 stars. To find out if the 2012 A3 is the right car for you, check out the pros and cons, trims, specs and options at Kelley Blue Book.";
```

Comment:
I think it could be used with data driven or not

Reply:
I remember Mr. Nam said that we shouldn't use data-driven for this